nemo/collections/tts/torch/g2ps.py

```diff
@@ -67,6 +70,7 @@ def __init__(
             ignore_ambiguous_words: Whether to not handle word via phoneme_dict with ambiguous phoneme sequences. Defaults to True.
             heteronyms (str, Path, List): Path to file with heteronyms (every line is new word) or list of words.
             encoding: Encoding type.
+            phoneme_probability (Optional[float]): The probability (0.<var<1.) that each word is phonemized. Defaults to None which is the same as 1.
```

Comment:
The probability (0.<var<1.) that each word is phonemized. -> The probability (0.<var<1.) that each word is phonemized if it is possible.

Because this tokenizer returns chars too.

Reply:
Updated comment